bee-bundle/src/transaction.rs

```diff
@@ -358,7 +300,6 @@ pub enum TransactionBuilderError {
     InvalidValue(i64),
 }
 
-#[derive(Default)]
```

Comment:
Why are we removing this ?

Reply:
Don't we agree on removing Default trait?

Reply:
On Transactions, not on Builders